==================================== INPUT ====================================
# expected-to-throw

fragment refetchableConflictWithOperationF on Node
  @refetchable(queryName: "refetchableConflictWithOperationQuery")
{
  __typename
}

query refetchableConflictWithOperationQuery {
  node(id: "y") {
    __typename
  }
}
==================================== ERROR ====================================
✖︎ A unique query name has to be specified in `@refetchable`, an operation `refetchableConflictWithOperationQuery` already exists.

  refetchable_conflict_with_operation.invalid.graphql:4:27
    3 │ fragment refetchableConflictWithOperationF on Node
    4 │   @refetchable(queryName: "refetchableConflictWithOperationQuery")
      │                           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    5 │ {

  ℹ︎ an operation with that name is already defined here

  refetchable_conflict_with_operation.invalid.graphql:9:7
    8 │ 
    9 │ query refetchableConflictWithOperationQuery {
      │       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
   10 │   node(id: "y") {
